addons/assigngear/functions/fnc_gui_vehicleGear_save.sqf

```diff
@@ -0,0 +1,22 @@
+#include "\x\tmf\addons\assignGear\script_component.hpp"
+/*
+ * Name = TMF_assignGear_fnc_vehicleGear_save
+ * Author = Head
+ *
+ * Arguments:
+ * None.
+ *
+ * Return:
+ * Data to store
+ *
+ * Description:
+ * Internal Use Only
+ */
+private _data = (missionNamespace getVariable [QGVAR(vehicleGear_data), ['', '', []]]);
+TRACE_1('SAVE', _data);
+private _gear = (_data # 2);
+{
```

Comment:
Appreciate you might be going for readability here but `apply is pretty neat.
`_gear = _gear apply {_x select (_x # 1) > 0};`

Reply:
Good catch! Simplified the code was a remnant from some more in-depth logic before I removed it